src/LibraryViewExtensionWebView2/Packages/Library/README.md

```diff
@@ -0,0 +1,308 @@
+# librarie.js
+Reusable lightweight library component based on React.js
```

Comment:
why is it necessary to commit the librarieJS package?

Reply:
Yes, we shouldn't need to commit the whole library js package into dynamo. Enzo is working on moving these changes to the library repo.